.jscsrc

```diff
@@ -1,51 +1,78 @@
 {
-    "disallowEmptyBlocks": true,
-    "disallowMixedSpacesAndTabs": "smart",
-    "disallowNewlineBeforeBlockStatements": true,
-    "disallowSpaceAfterObjectKeys": true,
-    "disallowSpaceBeforePostfixUnaryOperators": true,
-    "disallowSpacesInFunction": {
-        "beforeOpeningRoundBrace": true
-    },
-    "disallowSpacesInCallExpression": true,
-    "disallowSpacesInsideArrayBrackets": true,
-    "disallowSpacesInsideParentheses": true,
-    "disallowTrailingComma": true,
-    "disallowTrailingWhitespace": true,
-    "requireBlocksOnNewline": 1,
-    "requireCamelCaseOrUpperCaseIdentifiers": true,
-    "requireCapitalizedConstructors": true,
-    "requireCommaBeforeLineBreak": true,
-    "requireCurlyBraces": true,
-    "requireDotNotation": true,
-    "requireLineBreakAfterVariableAssignment": true,
-    "requirePaddingNewlinesBeforeKeywords": true,
-    "requireParenthesesAroundIIFE": true,
-    "requireSpaceAfterBinaryOperators": true,
-    "requireSpaceAfterKeywords": [
+    "requireCurlyBraces": [
         "if",
         "else",
         "for",
         "while",
         "do",
-        "switch",
-        "case",
-        "return",
         "try",
         "catch"
     ],
-    "requireSpaceAfterLineComment": true,
-    "requireSpaceBeforeBinaryOperators": true,
-    "requireSpaceBeforeBlockStatements": true,
-    "requireSpaceBeforeKeywords": [
-        "else",
-        "while",
-        "catch"
+    "requireOperatorBeforeLineBreak": true,
+    "requireCamelCaseOrUpperCaseIdentifiers": true,
+    //"maximumLineLength": {
```

Comment:
I'd tend to agree; we turned this rule off on the Miyagi project but I can see there were a few "ridiculous" cases where really it should have been enforced. I'd not be opposed to putting something on the ruleset just to get us started. Perhaps a hundred characters or something close to this? /cc @njmcode